llama_stack/cli/model/list.py

```diff
@@ -47,15 +54,28 @@ def _run_model_list_cmd(self, args: argparse.Namespace) -> None:
                 continue
 
             descriptor = model.descriptor()
-            rows.append(
-                [
-                    descriptor,
-                    model.huggingface_repo,
-                    f"{model.max_seq_length // 1024}K",
-                ]
+            if args.search:
```

Comment:
I think this code can be slightly improved in legibility by combining the blocks:

```python
if not args.search or args.search.lower() in descriptor.lower():
   rows.append(...)
```